bitnami/minio/templates/prometheusrule.yaml

```diff
@@ -9,11 +9,8 @@ kind: PrometheusRule
 metadata:
   name: {{ include "common.names.fullname" . }}
   namespace: {{ default .Release.Namespace .Values.metrics.prometheusRule.namespace | quote }}
-  labels: {{- include "common.labels.standard" . | nindent 4 }}
+  labels: {{- include "common.labels.standard" ( dict "customLabels" .Values.commonLabels "context" $ ) | nindent 4 }}
     app.kubernetes.io/component: metrics
-    {{- if .Values.commonLabels }}
-    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
-    {{- end }}
     {{- if .Values.metrics.prometheusRule.additionalLabels }}
```

Comment:
Use `merge`

Reply:
I'm only merging when there's a `.Values.XXX.labels` parameter and skipping it when the parameter name is `.Values.XXX.additionalLabels` since the naming suggests they should be added (not merged)